Cargo.toml

```diff
@@ -181,20 +181,24 @@ walkdir = "2"
 xsalsa20poly1305 = "0.9"
 zstd-safe = "6.0.5"
 
+# hamtv0 specific deps
+forest_hash_utils = "0.1"
```

Comment:
This needs to go.

Reply:
Replace with? `forest_hash_utils` is required even for the existing fvm 3.0's hamt impl by `fvm_ipld_hamt` crate (https://github.com/filecoin-project/ref-fvm/blob/3ad5b90f35085e26585e47674844bb72ef09fb06/ipld/hamt/Cargo.toml#L18).

Reply:
That is most tragic. The `forest_hash_utils` crate is dead and gone. We either have to upstream a fix or move forward with our own `hamt` crate.